ci/scripts/deterministic-e2e-test.sh

```diff
@@ -25,3 +25,10 @@ timeout 90s cargo make sslt --release -- -j 16 './e2e_test/streaming/**/*.slt'
 
 echo "--- deterministic simulation e2e, ci-3cn-2fe, parallel, batch"
 timeout 30s cargo make sslt --release -- -j 16 './e2e_test/batch/**/*.slt'
+
+# bugs here!
```

Comment:
👀👀👀

Reply:
Do we have an issue for this?

Reply:
Not yet. I'm confirming it and will open one once I find the cause.